addons/zeus/functions/fnc_ui_removeObjects.sqf

```diff
@@ -0,0 +1,65 @@
+/*
+ * Author: Fisher
+ * Removes all objects in given radius for all curators.
+ *
+ * Arguments:
+ * 0: Dummy controls group <CONTROL>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+ 
+#include "script_component.hpp"
+ 
+disableSerialization;
```

Comment:
These UI initialisation functions will stem from an `onSetFocus` event handler, which I don't believe is scheduled, not sure though.